src/pages/Register.tsx

```diff
@@ -1,5 +1,62 @@
+// import { useTranslation } from "react-i18next";
+import { Box, Typography, styled, TextField, Paper } from "@mui/material";
 export default function Register() {
+  const PaperStyled = styled("div")`
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  `;
+
   return (
-    <div>Register</div>
-  )
+    <Box
+      component="section"
+      width="50vw"
+      sx={{ display: "flex", position: "absolute", top: "10%" }}
+    >
+      <PaperStyled>
+        <Paper elevation={3}>
+          <Typography variant="h3" color="initial" justify-content=" center">
+            Registrate
+          </Typography>
+          <Box
+            component="form"
+            sx={{
+              "& .MuiTextField-root": { m: 1, width: "25ch" },
+            }}
+            noValidate
+            autoComplete="off"
+          >
+            <div>
+              <TextField
+                required
+                id="outlined-required"
+                label="Nombre"
```

Comment:
Same here

And then change the text using the react-i8next format like:
`{t("register-page.label-name")}`

add this new text on the translation.json file that is already added on the public/locales/es
Should be something like: 
```
"register-page": {
    "label-name": "Nombre",

  }
```